.github/workflows/ci-test-backend.yml

```diff
@@ -61,7 +55,8 @@ jobs:
         working-directory: backend/_example/memory_store
 
       - name: submit coverage
```

Comment:
what the reason to change it? it doesn't make code shorter and more readable but adds another dependency for no good reason.

Reply:
Because this step looks like as some that can be replaced on public action with configuration params.

Reply:
I still think this is unnecessary dependency. In addition, it doesn't work or works differently. I don't see your builds on https://coveralls.io/github/umputun/remark42

Reply:
Job #108898c84-PR-632.0
https://coveralls.io/jobs/60912751

Reply:
which shows "PENDING COMPLETION", whatever it means. Maybe coveralls is slow, not sure

Reply:
Might be related to parallel flag?

Reply:
I think that it's related with fact that PR from fork repo to original

Reply:
@umputun @paskal do you want that I will revert old definition for this step?

Reply:
I am OK with that only if it works. This action is run by CI in this repo and there shouldn't be problems with a fork: fork results would go into fork coveralls account.
Can you please try to make it work first? If you'll enable coveralls for your own account, your fork results should appear there.

Reply:
https://coveralls.io/jobs/61033004 build in my forked repo

Reply:
https://coveralls.io/jobs/61033012 last build for PR here

Reply:
@Smolevich problem is it's still in `pending completion` state. I ask again, could be it because of a parallel flag? What's the point of having it?

Reply:
Other PR in Coveralls example:
<img width="954" alt="image" src="https://user-images.githubusercontent.com/712534/78538176-e9683b00-77f0-11ea-9ba9-3934cf9c8b70.png">
I don't see a single coveralls run assigned to this PR (e.g. a line with `PULL #632`), in [coveralls builds](https://coveralls.io/repos/289736/builds) page so even if it works it still loses PR annotation. My point is still the same: either fix it or remove it, but in the current state of this PR, something is not right.

Reply:
[It seems to work](https://coveralls.io/builds/29871429) after the last commit: I was right to be suspicious about the parallel flag. I am OK with switching to this.

Reply:
I gave it some more thought and I tend to agree with @umputun : we're introducing some magic here which is less transparent than plain shell code (but does run some shell code underneath) and gives us no benefits.

Reply:
Theoretically, I can see how this can be useful in the form of universal better-go-actions including inside everything needed to build, test, lint and make/submit coverage reports. However our flow, even with this piece, still half-declarative and half-imperative, so I struggle to see how this change will make anything better.

Reply:
Action with coveralls instead shell script can be useful if this functional is needed more than one repository 
I agree that we don't have benefit for current case

Reply:
@Smolevich could you then please revert goveralls change and leave only other changes with code clarification? Also there might some small things to fix after the latest changes, like a typo in `tast and lint examples`. Thank you!

Reply:
@paskal i revert changes and fix typo

Reply:
@umputun, are you agree with changes?